pkg/cli/cmd/radinit/cluster.go

```diff
@@ -39,11 +39,11 @@ func (r *Runner) enterClusterOptions(options *initOptions) error {
 	if err != nil {
 		return clierrors.MessageWithCause(err, "Unable to verify Radius installation.")
 	}
-	options.Cluster.Install = !state.Installed
+	options.Cluster.Install = !state.RadiusInstalled || !state.DaprInstalled
```

Comment:
Does this mean that if **Dapr** is not installed we can't run `rad init`?

Reply:
Do you mean if radius is installed but dapr is not?

Reply:
Got my answer to my question :) Thanks @superbeeny!